ext/priority.d.ts

```diff
@@ -0,0 +1,8 @@
+declare const priority: {
+  USER_REJECT: number
+  AUTO_REJECT: number
+  AUTO_KEEP: number
+  USER_KEEP: number
+}
```

Comment:
All of these should be marked as deprecated.

Reply:
See above.